packages/webdriver/protocol/saucelabs.json

```diff
@@ -108,6 +108,29 @@
       }]
     }
   },
+  "/session/:sessionId/sauce/ondemand/throttle/cpu": {
```

Comment:
@christian-bromann does this returns something?

Reply:
nope